parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -132,7 +122,7 @@ ensureAuditD() {
     systemctlEnableAndStart auditd || exit $ERR_SYSTEMCTL_START_FAIL
   else
     if apt list --installed | grep 'auditd'; then
-      systemctlDisableAndStop auditd || exit $ERR_SYSTEMCTL_START_FAIL
+      apt_get_purge 20 30 120 auditd &
```

Comment:
Removing the package I assume does `systemctl stop` as well?

Reply:
It removes the systemd service definition altogether, so yeah explicitly stopping/disabling on startup is no longer a thing. (Also this was the only use of that func hence it goes altogether)